include/mbedtls/platform_util.h

```diff
@@ -238,6 +238,21 @@ int mbedtls_platform_memcmp( const void *buf1, const void *buf2, size_t num );
  */
 uint32_t mbedtls_platform_random_in_range( size_t num );
 
+/**
+ * \brief       Random delay function.
+ *
+ *              Function implements random delay by incrementing local variable
+ *              randomized number of times.
+ *
+ * \note        Currently the function is dependent of hardware providing an
+ *              rng with MBEDTLS_ENTROPY_HARDWARE_ALT.
+ *
+ * \param num   Max-value for the local variable increments.
+ *
+ * \return      In success number of increments, -1 in case of errors.
+ */
+int mbedtls_platform_random_delay( size_t num );
```

Comment:
Should there be a default value for num? Or at least some hint how much delay does this generate with different num values?

Reply:
I tried to rephrase the function comments. 
The purpose of this function is to generate a random delay, number of variable increments can be anything between 1 to `num`. I feel that "default value" is not suitable for function that is random by nature.

Function caller should not use this function to create a predictable delay. There should be other, more sophisticated methods for generating deterministic delay.